### PR TITLE
Ton of MI updates

### PR DESCRIPTION
**Pull request by andrew-sotiriou**
#### What does this PR do?

Updates to React-GPT so we can fire an Interstitial

#### How can this change be manually tested?

use the test pages noted in the Oxygen PR

#### Asana Card

https://app.asana.com/0/1198498632694766/1200470002742049/f

#### Questions or concerns about this change?

n/a

#### Deployment Notes

n/a

---

#### PR Readiness

* [x] Changes have been validated in local dev

#### Code Readiness

* [x] Code follows [AT Media styles and conventions](https://apartmenttherapy.slite.com/app/channels/R54jSBx0aG/notes/RCwBYjgA4B)
* [x] Code has been [auto-formatted and is free of lint errors](https://apartmenttherapy.slite.com/app/channels/R54jSBx0aG/notes/BXnBJVigtW)
* [x] Unrelated code changes are not present
* [x] Relevant inline documentation or READMEs have been updated
* [x] Application and browser logs reviewed for runtime warnings and errors
* [x] Correct level of [logging](https://apartmenttherapy.slite.com/app/channels/R54jSBx0aG/notes/gwTZjrJhOy) added to code

#### Automated Testing

* [x] [Unit tests](https://apartmenttherapy.slite.com/app/channels/R54jSBx0aG/notes/k6Uah8Iquw) are written at the appropriate level given the code’s risk-level
* [x] Existing tests still pass
* [x] No unnecessary test runner output, e.g. debugging or warnings are not present in Jest or Unittest output

#### Other Considerations

* [x] Relevant documentation updated in Slite

Also consider this PR's impact on architecture and security; AMP, Apple News, RSS; SEO, GA, performance and accessibility.

### CODE REVIEW

File: src/Bling.js [hunk 1]
```diff
@@ -797,6 +800,10 @@ class Bling extends Component {
             this._adSlot = null;
         }
         this._divId = id || Bling._adManager.generateDivId();
+        
+        if (this.props.outOfPage) {
+            return <Fragment />;
```
Comment (andrewb) [inline src/Bling.js]: `return null` would be more idiomatic

**Review by andrewb — APPROVED** [review]
Thanks, one minor comment

File: src/Bling.js [hunk 2]
```diff
@@ -642,7 +642,8 @@ class Bling extends Component {
             if (outOfPage) {
                 this._adSlot = Bling._adManager.googletag.defineOutOfPageSlot(
                     adUnitPath,
-                    divId
+                    Bling._adManager.googletag.enums.OutOfPageFormat
+                        .INTERSTITIAL
```
Comment (Engywook2005) [inline src/Bling.js:647]: Add comment here that we may need additional logic when we have other OOP types?

**Review by Engywook2005 — APPROVED** [review]
Approved with one house-keeping suggestion

File: src/Bling.js (hunk 2 above)
Comment (andrew-sotiriou) [reply src/Bling.js:647]: added

File: src/Bling.js (hunk 1 above)
Comment (andrew-sotiriou) [reply src/Bling.js]: updated...may need to revert if it breaks something

**Pull request merged**